lib/main/atom/fileStatusCache.ts

```diff
@@ -11,6 +13,7 @@ export interface FileStatus {
 
 let fileStatuses: { [index: string]: FileStatus } = {};
 export function getFileStatus(filePath: string): FileStatus {
+    filePath = path.normalize(filePath);
```

Comment:
Prefer `consistentPath` https://github.com/TypeStrong/atom-typescript/blob/master/lib/main/utils/fsUtil.ts#L5-L8 i.e. `/` *everywhere*. This is what the TypeScript compiler does *internally*. We have a bit of a dance as atom works one way, and typescript the other.